content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -719,9 +719,13 @@ is turned on.
   more than one default is specified, the admission plugin forbids the creation of
   all PVCs.
 * If the admission plugin is turned off, there is no notion of a default
-  StorageClass. All PVCs that have no `storageClassName` can be bound only to PVs that
-  have no class. In this case, the PVCs that have no `storageClassName` are treated the
-  same way as PVCs that have their `storageClassName` set to `""`.
+  StorageClass. All PVCs that have `storageClassName` set to `""` can be
+  bound only to PVs that have `storageClassName` also set to `""`.
+  However, PVCs with missing `storageClassName` can be updated later once
+  default StorageClass becomes available. If the PVC gets updated it will no
+  longer bind to PVs that have `storageClassName` also set to `""`.
+
+See [Retroactive StorageClass assignment](#retroactive-storageclass-assignment) for more details.
```

Comment:
In case `Retroactive StorageClass assignment` is not a feature gate, I prefer the following way.

```suggestion
See [retroactive StorageClass assignment](#retroactive-storageclass-assignment) for more details.
```